infrastructure/terraform.tfvars.tmpl

```diff
@@ -17,7 +17,7 @@ vector_secret_name="${VECTOR_SECRET_NAME}"
 vector_security_group="${VECTOR_SECURITY_GROUP}"
 vector_vpc="${VECTOR_VPC:-null}"
 workflow_root_path="${WORKFLOW_ROOT_PATH}"
-cloudfront_id="${VEDA_CLOUDFRONT_ID}"
 cognito_domain="${VEDA_COGNITO_DOMAIN}"
 client_id="${VEDA_CLIENT_ID}"
 userpool_id="${VEDA_USERPOOL_ID}"
+disable_default_apigw_endpoint="${VEDA_DISABLE_DEFAULT_APIGW_ENDPOINT:-false}"
```

Comment:
```suggestion
```

may not be necessary, update secret to have `TF_VARS_DISABLE_DEFAULT_APIGW_ENDPOINT`